test/batches_SUITE.erl

```diff
@@ -303,3 +306,93 @@ handle_call(N, _From, Acc) ->
 
 handle_cast(_Msg, Acc) ->
     {noreply, Acc}.
+
+%% Retry test
+
+retry_request(_) ->
```

Comment:
I'd move new test cases right bellow the other ones and new helpers to other helpers similarly. It would be more readable then.